[Style Guide] Content Strategy (#9846)
cloudflare/cloudflare-docs: content/style-guide/content-strategy/documentation-content-strategy/content-types/3rd-party-integration-guide.md

---
pcx_content_type: concept
title: 3rd-party integration guide

---

> # 3rd-party integration guide

Reviewer: 3rd and third are both used, perhaps we can stick to just one

<table>
  <tr>
    <th style="width:20%">Purpose</th>
    <th>The purpose of a third-party integration guide is to explain how to use a third-party product with Cloudflare.</th>
  </tr>
  <tr>
    <td>Tone</td>
    <td>instructional, straightforward</td>
  </tr>
  <tr>
    <td>content_type</td>
    <td>integration-guide</td>
  </tr>
  <tr>
    <td>Required components</td>
    <td>Title <br/>Context <br/>Prerequisites <br/>Steps <br/>Links</td>
  </tr>
  <tr>
    <td>Optional components</td>
    <td>Notes/warnings <br/> Examples <br/> Screenshots <br/> Tables <br/> Step validation</td>
  </tr>
</table>

> Though we do want to help our customers and integrations between different products can definitely be a pain point, there is a large risk/maintenance cost associated with specific types of 3rd-party resources.

Reviewer: ```suggestion
Although we want to help our customers as integrations between different products can be a pain point, there is a large risk and maintenance cost associated with specific types of 3rd-party resources.
```

## Structure

### Required components

> + [**Title**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/titles/): Short verb phrase in second-person imperative that includes the third-party name. Do not use gerund phrases.

Reviewer: same comment I left from the section on API content types regarding indent/bullets

+ [**Steps**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/steps-tasks-procedures/): Numbered steps that complete a task.
  + Proceed with caution: creating step-by-step instructions of 3rd-party environments
    + Discouraged generally, but acceptable in certain situations. General preference is to link back to an article that someone else maintains.
    + Easily become out-of-date, especially if we can not access the 3rd-party product
  + Link out for basic concepts (Regex, JavaScript, web server maintenance).
+ [**Links**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/links/): May be a bulleted list that references the third-party product or in-text links to the third-party process documentation.
  + Link to reputable sources within reason.

### Optional components

+ [Notes/warnings](/style-guide/content-strategy/documentation-content-strategy/component-attributes/notes-tips-warnings/)
+ [Examples](/style-guide/content-strategy/documentation-content-strategy/component-attributes/examples/)
+ Screenshots

>   + Screenshots of the third-party product are highly discouraged. It is all the problems of video/screenshot maintenance, but with a much greater risk that something changes and we are not aware of it.
>   + May become an even bigger problem if we can not access the 3rd-party product
> 

Reviewer: might be good to add this as an aside
```suggestion
  + Screenshots of the third-party product are highly discouraged. It has all the problems of video or screenshot maintenance, but with a much greater risk that something changes and we are not aware of it.
  + It may become an bigger problem if we can not access the 3rd-party product.

```
